scripts/test_wrapper

```diff
@@ -22,6 +20,7 @@ ENDXML
   exit 0
 else
   # Lint failed
+  >&2 echo $LINTER_OUTPUT
```

Comment:
```suggestion
  echo $LINTER_OUTPUT >&2 
```
Not sure if it's just my problem, but I haven't known the syntax with redirect before command. To me, redirects after command look more comprehensible.